react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,40 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+JavaScript library for building user interfaces
+- What is a React component?
+reusable piece of UI that summaries its own logic and rendering
+- How do you mount a React app (root component) to the DOM?
+use `ReactDOM.render()`
```

Comment:
Use the react-dom package to create a root attached to an HTML element (usually a div)
Call root.render() and pass it the app's main component.